documentation/gitbook/how-to-run/multi-configs.md

```diff
@@ -0,0 +1,45 @@
+---
+description: >-
+  How to use multiple configuration files.
+---
+
+# Using Multiple Configuration Files
```

Comment:
Maybe make this title more descriptive - "Patching together multiple configuration files"